File: FizzBuzz.java
class FizzBuzz {
	private static boolean byThree(int a) {
		return (a % 3) == 0;
	}
	
	private static boolean byFive(int a) {
		return (a % 5) == 0;
	}
	
	public static void main(String[] args) {
		for (int i = 1; i < 101; i++) {
			if (!byThree(i) && !byFive(i)) {
				System.out.print(i);
			} else {
				if (byThree(i)) {
					System.out.print("Fizz");
				}
				if (byFive(i)) {
					System.out.print("Buzz");
				}
			}
			System.out.print("\n");
		}
	}
}

File: fizzbuzz.rb
(1..100).each do  |a| 
  unless (a % 3 == 0  || a % 5 == 0) 
    print a
  else
    print 'Fizz' if a % 3 == 0; 
    print 'Buzz' if a % 5 == 0; 
  end
  print "\n"
end